reana_commons/config.py

```diff
@@ -226,6 +226,12 @@ def kubernetes_node_label_to_dict(node_label):
 }
 """REANA Workflow Controller address."""
 
+REANA_WORKFLOW_SCHEDULING_POLICIES = ["fifo", "balanced"]
+"""REANA workflow scheduling policies.
+- ``fifo``: first-in-first-out strategy regardless of users.
+- ``balanced``: multi-user-aware strategy.
```

Comment:
balanced: a weighted strategy taking into account existing multi-user workloads and the DAG complexity of incoming workflows


... would be more precise, albeit a bit longer.